### ISSUE TITLE
[Bug Report] multiple loading of the same styles using custom theme

### ISSUE THREAD

**Issue by okosowski** [ElemeFE/element#15266]
### Element UI version
2.7.2, 2.8.0

### OS/Browsers version
osx mojave 10.14.4

### Vue version
2.6.10

### Reproduction Link
https://github.com/okosowski/vue-element-io-issue/tree/custom

### Steps to reproduce
using `Reproduction Link` - download repo, install depemdency, run project by npm start. You can see in browser that styles for el-button is loaded 4 times.

Using own vue project, add elementio and custom theme base on https://element.eleme.io/#/en-US/component/custom-theme

### What is Expected?
loading .el-button (and any other style only once, like it is importing default .css style)

### What is actually happening?
loading the same .el-button styles 4 times

**Comment by okosowski** [comment]
example of multiple loading .el-button style:
![Zrzut ekranu 2019-04-25 o 09 51 52](https://user-images.githubusercontent.com/6871271/56721569-71bb1080-6745-11e9-8e4f-b4075b05e1dd.png)

**Comment by JBtje** [comment]
duplicate of #11817

**Comment by okosowski** [comment]
duplicate of #11817
thanks @JBtje
